nextroute/model_latestend.go

```diff
@@ -18,6 +18,11 @@ type LatestEnd interface {
 
 	// Lateness returns the lateness of a stop. The lateness is the difference
 	Lateness(stop SolutionStop) float64
+
+	// SetLatenessFactor adds a factor with which a deviating stop is
+	// multiplied. This is only taken into account if the construct is used as
+	// an objective.
+	SetLatenessFactor(factor StopExpression)
```

Comment:
I would change this to such that you can check factor >= 0
`SetFactor(factor float64, stop StopExpression) ` 
and also introduce a so one can always verify what has been used:
`Factor(stop StopExpression) float64`

Reply:
changed